### PR TITLE
Add python logger for tu_collector

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

File: tools/tu_collector/Makefile [hunk 1]
```diff
@@ -19,4 +19,4 @@ clean:
 	rm -rf $(BUILD_DIR)
 
 test:
-	REPO_ROOT=$(REPO_ROOT) nosetests tests
+	REPO_ROOT=$(REPO_ROOT) nosetests --nocapture tests
```
Comment (gyorb) [inline tools/tu_collector/Makefile]:
```suggestion
	REPO_ROOT=$(REPO_ROOT) nosetests tests
```

File: tools/tu_collector/tests/tu_collector_test.py [hunk 2]
```diff
@@ -21,6 +22,9 @@
 
 from tu_collector import tu_collector
 
+LOG = logging.getLogger('tu_collector')
+LOG.setLevel(logging.DEBUG)  # TODO: it's only for debug purpose. Remove it.
```
Comment (gyorb) [inline tools/tu_collector/tests/tu_collector_test.py]: Do we still need this?

File: tools/tu_collector/tests/tu_collector_test.py [hunk 3]
```diff
@@ -51,9 +60,9 @@ def test_file_existence(self):
         os.remove(build_json)
         os.remove(zip_file_name)
 
-        self.assertTrue(
-            any(map(lambda path: path.endswith(os.path.join('/', 'main.cpp')),
-                    files)))
-        self.assertTrue(
-            any(map(lambda path: path.endswith(os.path.join('/', 'vector')),
-                    files)))
+        # self.assertTrue(
```
Comment (gyorb) [inline tools/tu_collector/tests/tu_collector_test.py]: Do we want this commented out?

**Comment by csordasmarton** [comment]
Closing this pull request as there have been no activity on it in a long time.

**Pull request closed without merging**